scripts/update/Updater.php

```diff
@@ -92,5 +93,12 @@ public function update($initialVersion)
         }
 
         $this->skip('0.12.0', '0.13.0');
+
+        if ($this->isVersion('0.13.0')) {
+            $setup = new SetupDecryptDeliveryLogFormatterService();
```

Comment:
this script uses getServiceLocator, you need to propagate it before to use.
Also you can use `$setup()` or `call_user_func($setup, [])` to be sexier

Reply:
I fix it, I don't like sexier I like to be explicit :nerd_face: